addon/components/au-date-picker.gts

```diff
@@ -11,29 +10,40 @@ import { guidFor } from '@ember/object/internals';
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { modifier } from 'ember-modifier';
+import AuLabel from './au-label';
+import type { DuetDatePickerChangeEvent } from '@duetds/date-picker/dist/types/components/duet-date-picker/duet-date-picker';
+import type { DuetLocalizedText } from '@duetds/date-picker/dist/types/components/duet-date-picker/date-localization';
+import type { DuetDateAdapter } from '@duetds/date-picker/dist/types/components/duet-date-picker/date-adapter';
 
-const props = modifier(
-  function props(element, positional, properties) {
-    for (let propertyName in properties) {
-      element[propertyName] = properties[propertyName];
-    }
-  },
-  { eager: false },
-);
+type IsoDate = string;
+type Adapter = DuetDateAdapter;
+type Localization = DuetLocalizedText;
 
-const DEFAULT_ADAPTER = {
-  parse: function (value = '', createDate) {
-    let dateRegex = /^(\d{1,2})-(\d{1,2})-(\d{4})$/;
+export interface AuDatePickerSignature {
+  Args: {
+    alignment?: 'top';
+    adapter?: Adapter;
+    buttonLabel?: string;
+    disabled?: boolean;
+    error?: boolean;
+    'first-day'?: string;
```

Comment:
We should probably support a `firstDay` argument as well, not sure why we went with a dasherized name in the first place.